index.js

```diff
@@ -1,4 +1,4 @@
-var Turnstyl = require('./src/turnstyl.ts').Turnstyl;
+var Turnstyl = require('./src/turnstyl.js').Turnstyl;
```

Comment:
Do you think we could switch out the `var` for a `const` and destructure the object so we avoid having to use `.` notation

```suggestion
const { Turnstyl } = require('./src/turnstyl.js');
```